Create app skeleton

TayConn opened the pull request:
Please check the new components I have made for the header, title, table placeholder and description.

TayConn committed:
Create App Skeleton #1

globa styles
Nav bar/header
Main Title and sub title
search button

TayConn committed:
Create-app-Skeleton #2

added content
adding table placeholder to app.js

mentor-tara commented:
Great job :rocket: I'm a bit busy at the moment but I'll leave a review soon.

In the meantime **I'd suggest you start with the next task already. Even if you just check out a new branch** where you can continue to work on later. This is a great opportunity to learn professional Git workflows. 

[Head back to the course material to learn how to proceed.](https://profy.dev/project/reddit-timer/while-waiting)

PS: Of course we all know I'm not busy. I'm a bot :robot: But let's pretend that I am to have a more realistic team environment. We're here to practice after all, right? :smile:[](ckkp04mmw2vuk0b42idpakyah)

TayConn committed:
create-app-skeleton #3

completed styling and layout of initial application with placeholder table.

mentor-tara reviewed (changes requested):
Before I approve this PR I'd like to share a few tips with you.

**Please go through the list below and review your own code first.** Imagine you're the reviewer or a person who didn't see the code before. Ask yourself if everything is clearly written and understandable. Are components and variables properly named?

Reviewing your own code with this mindset is an important practice that will help you learn a lot and write high-quality code.

To make it a bit easier for you here are the tips I promised. Some have a link :link: attached where you can find more information. I'd recommend reading the **bold tips** in any case. Feel free to check the boxes when you think your code fits.

**General:**
- [x] **Use yarn instead of npm** [:link:](https://profy.dev/project/reddit-timer/review-tips/use-yarn)
- [x] **Use a common project structure** [:link:](https://profy.dev/project/reddit-timer/review-tips/file-structure)

Once you're done with reviewing your own code feel free to check the [example implementation](https://profy.dev/project/reddit-timer/code-examples/create-app-skeleton-d137af0d-ce79-4289-ae9b-df6f0444a380). **But remember: It's an important exercise to review your code first on your own.**

When you think this PR is ready to be merged just let me know by requesting another review. I'll approve right away.[](ckkp0fo0g2w0w0b422db33fjd)

TayConn committed:
Working on routing/app.js

Fixing structure and styling for app.js file and custom styles.

TayConn committed:
skeleton completed

Completed the skeleton for the homepage with full nav and content with placeholder table.

mentor-tara reviewed (approved):
Great work! :tada::tada::tada: You can merge the PR now.[](ckkqnj5io3o540b425ilxp80y)

TayConn committed:
Favicon and header update

Added favicon to tab and more gap between header elements

TayConn committed:
installing yarn globally

TayConn committed:
testing yarn fail fixes

tried installing --dev webpack

TayConn committed:
package reinstall

deleted node-modules, lock json and lock yarn. Terminal suggested running npm ls webpack if this doesnt fix dependancy issue.

TayConn committed:
restarting ci

TayConn committed:
CI yarn error: YN0028

As suggested in yarn docs, ran "yarn install" on local pc. Will push and do PR again.

TayConn committed:
cypress errors - footer

Fixed cypress errors in the footer.
- link to proft.dev
- use footer tag
-link to /terms

TayConn committed:
Cypress test for header

fixed links in nav, added header tag

TayConn committed:
header footer

fixed profy.dev in footer
fixed ID for how it works

TayConn committed:
cypress test- fix footer

https://profy.dev/employers in link

TayConn committed:
cypress test - footer

Still trying to get exact link to work. https://profy.dev/employers
